python/ray/exceptions.py

```diff
@@ -161,11 +161,17 @@ def __str__(self):
                 "https://ray.readthedocs.io/en/latest/memory-management.html"))
 
 
+class RayTimeoutError(RayError):
```

Comment:
RayTimeoutError(RayError, TimeoutError)

Reply:
`TimeoutError` is only python3-compatible